aiida-registry-app/src/App.css

```diff
@@ -1,11 +1,23 @@
-#root {
 
+#root {
```

Comment:
I think the first blank line is not needed ;)

Reply:
Yes, there are linting tools for javascript projects to automatically fix the coding style issues, I'll use one on another PR.